homeassistant/components/sensor/tellduslive.py

```diff
@@ -85,7 +85,9 @@ def name(self):
     @property
     def state(self):
         """Return the state of the sensor."""
-        if self._type == SENSOR_TYPE_TEMP:
+        if not self.available:
+            return STATE_UNKNOWN
```

Comment:
I think the constant is better while we can ever time change the value of `STATE_UNKNOWN`.